src/sketchup_objects/link_entities/spring_cylinder.rb

```diff
@@ -28,13 +30,40 @@ def change_color(color)
     @entity.material.alpha = 1.0
   end
 
-  def create_cylinder(center, up_vector, length, diameter, name)
-    definition = Sketchup.active_model.definitions.add(name)
-    circle_edgearray = definition.entities.add_circle(center,
-                                                      up_vector,
-                                                      diameter, 12)
+  def create_cylinder(center, up_vector, length, fix_plates_offset, spring_diameter)
+    segment_number = 24
```

Comment:
I think all constant in that file should be together at the top. (Where the RADIUS is)

Reply:
Done in de186d8